CMakeLists.txt

```diff
@@ -323,6 +323,15 @@ IF (KOKKOSKERNELS_ENABLE_TPL_CUBLAS)
   LIST(APPEND TPL_LIST "CUBLAS")
 ENDIF()
 
+# ==================================================================
+# Fortran Complex BLAS
+# ==================================================================
+
+IF (KOKKOSKERNELS_INTERNAL_ENABLE_HOST_BLAS)
+  INCLUDE(CheckHostBlasReturnComplex.cmake)
```

Comment:
@jjwilke adding you here to keep your work on #120 in sync with a couple updates that were made for CMake builds (and Makefile builds) to support testing and setting proper fortran mangling and APIs for proper BLAS TPL support.